VirtualCapability/VirtualCapability.php

```diff
@@ -260,6 +260,7 @@ class VirtualCapability_IsAppWebview extends VirtualCapability {
 		'MiuiBrowser',
 		'MQQBrowser',
 		'CriOS',
+	    'Firefox',
```

Comment:
Ok, no problem! I just wanted to make sure this comes from the original code & not from your own messing around ;)